### DISCUSSION TITLE
undefined method Column::components()

### DISCUSSION THREAD

**Discussion by larylinz** [rappasoft/laravel-livewire-tables#356] [Q&A] [answered]
Have i missed something?
I'm starting with this package, using bootstrap 5 ('theme' => 'bootstrap-5') and getting the error:

Call to undefined method Rappasoft\LaravelLivewireTables\Views\Column::components()

My Controller:

namespace App\Http\Livewire\Instructions;

use \App\Models\InstructionSheet;
use Illuminate\Database\Eloquent\Builder;
use Rappasoft\LaravelLivewireTables\DataTableComponent;
use Rappasoft\LaravelLivewireTables\Views\Column;

class InstructionsTable extends DataTableComponent
{

    public function columns(): array
    {
        return [

            Column::make('Name')
                ->searchable()
                ->sortable(),
            Column::make('Kunde', 'customer.name')
                ->searchable()
                ->sortable(),
            Column::make('erstellt', 'created_at')
                ->searchable()
                ->sortable(),
            Column::make('Actions')
                ->components([
                    Link::make('Edit'),
                ]),

            ];
    }
    
    public function query(): Builder
    {
        return InstructionSheet::query();
    }
    
}

within my Blade-File:

    <div class="row">
        <div class="col-md-12">
            @livewire('instructions.instructions-table')
        </div>
    </div>

Any help is appreciated!

**Comment by rappasoft** [comment, marked as answer]
Version 1 does not have components. They've been gone since April.

**Reply by larylinz** [reply]
OK, thank you for clarification!
So I will use 'format' with 'return view(...' or return '<a ...' with '->asHtml()' to make some Edit- or Delete-Buttons. Right?

**Reply by rappasoft** [reply]
Yes or a [custom row](https://github.com/rappasoft/laravel-livewire-tables/wiki/Custom-Row-View). But the first should work fine.

**Reply by larylinz** [reply]
Thank you!!
